tests/suite_utils.py

```
@@ -201,15 +201,22 @@ def stress_worker():
-            g.client.execute_command("BGREWRITEAOF", target_nodes="primaries")
+            try_bgrewriteaof(g.client, target_nodes="primaries")
     else:
         g.client.execute_command("FLUSHALL")
-        g.client.execute_command("BGREWRITEAOF")
+        try_bgrewriteaof(g.client)
+
+
+def try_bgrewriteaof(client, **kwargs):
+    try:
+        client.execute_command("BGREWRITEAOF", **kwargs)
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)